CHANGELOG.md

```diff
@@ -25,6 +25,8 @@ Usage:
 * Fix #460: Added a Quickstart for implementing and using a Custom Enricher based on Eclipse JKube Kit Enricher API
 * Fix #240: No Ports exposed inside Deployment in case of Zero Config Dockerfile Mode
 * Fix #480: wildfly-jar doesn't depend on common-maven module
+* Added a Quickstart for implementing and using a Custom Enricher based on Eclipse JKube Kit Enricher API
```

Comment:
This is duplicate of Line 25 `Fix #460: Added a Quickstart for implementing and using a Custom Enricher based on Eclipse JKube Kit Enricher API`